### PR TITLE
Add single clustering key scan integration test

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds an integration test of scan operation for single clustering key table. Please take a look!

### CODE REVIEW

File: core/src/integration-test/java/com/scalar/db/storage/StorageMultipleClusteringKeyScanIntegrationTestBase.java [hunk 1]
```diff
@@ -41,14 +36,13 @@
...
 import java.util.stream.IntStream;
-import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.Test;
 
 @SuppressFBWarnings(
     value = {"MS_CANNOT_BE_FINAL", "MS_PKGPROTECT", "ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD"})
-public abstract class StorageMultipleClusteringKeysIntegrationTestBase {
+public abstract class StorageMultipleClusteringKeyScanIntegrationTestBase {
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/StorageMultipleClusteringKeyScanIntegrationTestBase.java:45]: I renamed it from `StorageMultipleClusteringKeysIntegrationTestBase` to `StorageMultipleClusteringKeyScanIntegrationTestBase`. I also renamed the classes that extends this class in the same way.

File: core/src/integration-test/java/com/scalar/db/storage/TestUtils.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
+package com.scalar.db.storage;
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/TestUtils.java:1]: I moved the common logic to this class `TestUtils`.

File: core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoTestUtils.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.scalar.db.storage.dynamo;
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoTestUtils.java:1]: I also moved the Dynamo related common logic to this class `DynamoTestUtils`.

File: core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcTestUtils.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
+package com.scalar.db.storage.jdbc;
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcTestUtils.java:1]: I also moved the JDBC related common logic to this class `JdbcTestUtils`.

File: core/src/integration-test/java/com/scalar/db/storage/StorageSingleClusteringKeyScanIntegrationTestBase.java [hunk 5]
```diff
@@ -0,0 +1,675 @@
+package com.scalar.db.storage;
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/StorageSingleClusteringKeyScanIntegrationTestBase.java:1]: This is the main class in this PR and it's an integration test of scan operation for single clustering key table.

**Review by feeblefakie — DISMISSED** [review]
LGTM! Thank you!

**Comment by brfrn169** [comment]
Since it's just adding an integration test which is not very critical, I'll merge it.

@thongdk8 Please take a look when you have time.

**Pull request merged**